### PR TITLE
doc/man3/EVP_EncryptInit.pod: add EVP_CIPHER_CTX_rand_key

### PR DESCRIPTION
**Pull request by p-steuer**
Description taken from CHANGES file.

- [x] documentation is added or updated

### CODE REVIEW

File: doc/man3/EVP_EncryptInit.pod [hunk 1]
```diff
@@ -274,6 +275,10 @@ is not supported.
 EVP_CIPHER_CTX_ctrl() allows various cipher specific parameters to be determined
 and set.
 
+EVP_CIPHER_CTX_rand_key() generates a random key of the appropriate length based
+on the cipher context. The EVP_CIPHER can provide its own random key generation
```
Comment (richsalz) [inline doc/man3/EVP_EncryptInit.pod]: should add a sentence saying that key should b at least as large as the value returned by evp-xxxx.

Comment (p-steuer) [reply doc/man3/EVP_EncryptInit.pod]: Reformulated. Feel free to propose the final wording, if its not okay .

File: doc/man3/EVP_EncryptInit.pod [hunk 2]
```diff
@@ -274,6 +275,11 @@ is not supported.
 EVP_CIPHER_CTX_ctrl() allows various cipher specific parameters to be determined
 and set.
 
+EVP_CIPHER_CTX_rand_key() generates a random key of size
```
Comment (richsalz) [inline doc/man3/EVP_EncryptInit.pod]: I literraly meant 
```
B<Key> must point to a buffer at least as big as the value returned by
EVP_CIPHER_CTX_key_length().
```

Comment (p-steuer) [reply doc/man3/EVP_EncryptInit.pod]: ok , misunderstood. added this sentence to the original submission.

**Comment by dot-asm** [comment]
Merged. Thanks.

**Pull request closed without merging**